src/helpers/git_helpers/merge_helpers.sh

```diff
@@ -45,3 +45,13 @@ function commit_squash_merge {
 function undo_steps_for_merge {
   echo "reset_to_sha $(current_sha) hard"
 }
+
+
+function post_undo_steps_for_commit_squash_merge {
+  local current_branch_name=$(get_current_branch_name)
+  echo "checkout $current_branch_name"
```

Comment:
It is needed because the undo file contains `skip_current_branch_steps`, which would cause the `revert_commit` below to get skipped. The checkout is to avoid this.